### DISCUSSION TITLE
Run a custom script from a custom link (in background)

### DISCUSSION THREAD

**Discussion by tux-mania** [netbox-community/netbox#9596] [Q&A]
Hi all.

I have search a way to launch a custom script from a custom link.
The only way I found is by writing the script URL on the field Link URL.
The problem is: I would like the script to run as soon as I click on the custom link, and not sending me to the script page where I need then to click on 'RUN'.

Is there a way to do it?
Thanks.

**Comment by julianstolp** [comment]
I think this is not possible. You will have to write a plugin to do this.
